lib/icons-to-woff.js

```diff
@@ -21,10 +21,7 @@ module.exports = function createIconFont (fs, icons, options) {
       name: options.name,
       normalize: true,
       fontHeight: options.enforcedSvgHeight ? options.enforcedSvgHeight : undefined,
-      log: function () {},
-      error: /** @param {any} err */function (err) {
```

Comment:
The `error` property does not appear to be a supported option in `svgicons2svgfont`. I wasn't able to find a valid code path in that project that would call this function, so to support 100% test coverage, I opted to drop it here. The `.on('error')` callback on the stream below is sufficient for handling all errors raised by `svgicons2svgfont`.